pkgs/top-level/all-packages.nix

```
@@ -17254,9 +17254,7 @@ in
 
   powerline-go = callPackage ../tools/misc/powerline-go { };
 
-  powerline-rs = callPackage ../tools/misc/powerline-rs {
-    inherit (darwin.apple_sdk.frameworks) Security;
-  };
+  powerline-rs = callPackage ../tools/misc/powerline-rs { };
```

```suggestion
  powerline-rs = callPackage ../tools/misc/powerline-rs { 
      inherit (darwin.apple_sdk.frameworks) Security;
  };
```
and then add it to the buildInputs